src/store.js

```diff
@@ -0,0 +1,33 @@
+import { store } from 'reefjs'
+import { requestPagePreview } from './api'
+
+const wpStore = store( {
+	title: null,
+	lang: null,
+	targetId: null,
+	position: null,
+	data: null,
+	colorScheme: 'detect'
+}, {
+	setColorScheme( state, colorScheme ) {
+		state.colorScheme = colorScheme
+	},
+	trigger( state, targetId, position, title, lang ) {
+		state.title = title
+		state.lang = lang
+		state.targetId = targetId
+		state.position = position
+		state.data = null
+		requestPagePreview( lang, title, wpStore.receiveContent )
+	},
+
+	receiveContent( state, data ) {
+		state.data = data
+	},
+
+	close( state ) {
+		state.targetId = null
```

Comment:
when i click on wordmark, i am not sure why close is called here. 🤔 
<img width="374" alt="Screenshot 2024-04-03 at 13 50 33" src="https://github.com/wikimedia/wikipedia-preview/assets/2560096/00ce823a-90e7-4a4f-89aa-a0974e7bf440">

Reply:
Hmm I can't replicate that behavior, for me it's closing correctly when clicking the X and not the wodmark

Reply:
Thanks @medied for checking, i can't test it now, I will wait until this branch ready then test this again.